lib/m2ts/metadata-stream.js

```diff
@@ -92,22 +92,17 @@ var
       }
     },
     'T*': function(tag) {
-      var values;
       if (tag.data[0] !== textEncodingDescriptionByte.Utf8) {
         // ignore frames with unrecognized character encodings
         return;
       }
       
       // parse text field, do not include null terminator in the tag value
       // frames that allow different types of encoding contain terminated text [ID3v2.4.0 section 4.]
-      values = parseUtf8(tag.data, 1, tag.data.length).replace(/\0*$/, '').split('\0');
-
+      tag.value = parseUtf8(tag.data, 1, tag.data.length).replace(/\0*$/, '')
```

Comment:
I'm not sure if we agreed to return the unsplit value or the 1st element after splitting by `\0`?

Reply:
Either way works as long as it's clearly documented.
I think I was originally thinking that `tag.value === tag.values[0]`

Reply:
So we're good to go